scripts/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will run the pre-push checks in the given order
+# Ktlint
+#
+
+# this will run the ktlint check script
+bash kotlin-lint-check
```

Comment:
Does this actually work? It seems like in the setup we only copy the pre-push, not the lint check script.

Reply:
Why do we need to copy the lint check script into the hooks folder?
the flow is like:
1. setup will be exec by the contributor
2. copy the pre-push to hooks
3. pre-push file will run the scripts in the scripts folder

As per our discussion in the above comments, the scripts cane be executable externally also.

Reply:
Sorry, to be clear I don't think we should copy the scripts into the hooks folder. I'm actually asking if this setup works because the hook doesn't correctly reference the scripts folder. It's assuming that the kotlin-lint-check script is either in the same directly or somewhere else on the path. Am I missing something?

Reply:
ops, missed it. Done now.